lib/discourse_api/api/invite.rb

```diff
@@ -3,11 +3,36 @@ module DiscourseApi
   module API
     module Invite
       def invite_user(params = {})
-        post("/invites", params)
+        args = API.params(params)
+          .optional(
+            :email,
+            :skip_email,
+            :custom_message,
+            :max_redemptions_allowed,
+            :topic_id,
+            :group_ids,
+            :expires_at
+          ).to_h
+
+        post("/invites", args)
       end
 
+      # TODO: Deprecated. Remove after 20210506
       def invite_user_to_topic(params = {})
```

Comment:
@oblakeerickson how about this?

Reply:
Let's make the year 2022 ;)

Reply:
Other than that it looks good, I'll merge it in later after you update the date.

Reply:
nice catch😅